src/components/layout/Footer.tsx

```diff
@@ -1,18 +1,173 @@
+    },
+    {
+      label: 'footerLabels.support',
+      href: routes.support,
+      external: false,
+    },
+    {
+      label: 'footerLabels.terms',
+      // terms route is not implemented yet
+      // href: routes.terms,
+      external: false,
+    },
+  ],
+  [
+    {
+      label: 'footerLabels.confidentiality',
+      // gdpr route is not implemented yet
+      // href: routes.gdpr,
+      external: false,
+    },
+    {
+      label: 'footerLabels.gdpr',
+      // gdpr route is not implemented yet
+      // href: routes.gdpr,
+      external: false,
+    },
+  ],
+]
+
+const socialLinks = {
+  facebook: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ',
+  twitter: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ',
+  linkedin: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ',
+  youtube: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ',
+  instagram: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ',
+}
+
 export default function Footer() {
+  const { t } = useTranslation()
   const classes = useStyles()
-  return <footer className={classes.footer}></footer>
+  const { locale } = useRouter()
+
+  return (
+    <Container component="footer" maxWidth="xl" disableGutters>
+      <Grid container className={classes.footer}>
+        <Grid item xs={12} sm={8} md={6}>
+          <Link href={routes.index}>
+            <a>
+              <PodkrepiLogo locale={locale} size="large" variant="fixed" />
+            </a>
+          </Link>
+          <div className={classes.footer__socialContainer}>
+            <Link href={socialLinks.facebook}>
+              <a target="_blank">
+                <Facebook className={classes.footer__socialLink} />
+              </a>
+            </Link>
+            <Link href={socialLinks.twitter}>
+              <a target="_blank">
+                <Twitter className={classes.footer__socialLink} />
+              </a>
+            </Link>
+            <Link href={socialLinks.linkedin}>
+              <a target="_blank">
+                <LinkedIn className={classes.footer__socialLink} />
+              </a>
+            </Link>
+            <Link href={socialLinks.youtube}>
+              <a target="_blank">
+                <YouTube className={classes.footer__socialLink} />
+              </a>
+            </Link>
+            <Link href={socialLinks.instagram}>
+              <a target="_blank">
+                <Instagram className={classes.footer__socialLink} />
+              </a>
+            </Link>
+          </div>
+          <span className={classes.footer__copyrights}>{t('footerCopyrights')}</span>
+        </Grid>
+        <Grid item xs={12} sm={4} md={6} className={classes.footer__right}>
+          {footerItems.map((subarr, i) => (
+            <Grid key={Math.random() * i} item xs={12} md={4}>
+              <ul>
+                {subarr.map(({ label, href, external }, j) => (
+                  <li key={Math.random() * j} className={classes.footer__link}>
+                    <Link href={t(href)}>
```

Comment:
We better not translate the href directly

Reply:
Also we better not use `Math.random()` for the key as the list of links is finite - we can use

```tsx
<li key={`${i}-${j}`}>
```